src/gurt/debug.c

```diff
@@ -441,8 +442,10 @@ d_log_sync_mask(void)
 
 	/* load facility mask environment (D_LOG_MASK) */
 	log_mask = getenv(D_LOG_MASK_ENV);
-	if (log_mask != NULL)
+printf(" SAB: d_log_sync_mask:  log_mask: %s\n", log_mask);
+	if (log_mask != NULL){
```

Comment:
(style)  space required before the open brace '{'

Reply:
Fixed.